sycl/test-e2e/Adapters/queue_submit_mode.cpp

```diff
@@ -1,6 +1,9 @@
 // REQUIRES: gpu, level_zero
 // UNSUPPORTED: level_zero_v2_adapter
 // UNSUPPORTED-INTENDED: v2 adapter does support regular cmd lists
+// UNSUPPORTED: target-native_cpu
+// UNSUPPORTED-INTENDED: Currently mark Native CPU as unsupported, should be
+// investigated and tracked post team transfer

```

Comment:
Just in case someone stumbles upon this and thinks it is generally acceptable:

`UNSUPPORTED-INTENDED` is only for cases where there is no plans for the test to be supported by the disabled target. As stated in the comment, these will be amended by the adopting team in the near future. I will allow this due to the time constaints.